### ISSUE TITLE
How to run as daemon?

### ISSUE THREAD

**Issue by viniciusbo** [verdaccio/verdaccio#83]
I've tried to use https://github.com/terminalcloud/terminal-tools/blob/master/daemonize.sh script but this message is getting me stuck: `Failed to start verdaccio.service: Unit verdaccio.service failed to load: No such file or directory.`. I'm going to try `daemonize` (I mean this one http://software.clapper.org/daemonize/) as well.

So I was wondering how you guys daemonize Verdaccio process.

**Comment by viniciusbo** [comment]
The best solution I found so far is to daemonize it using PM2.

```sh
pm2 start /usr/local/lib/node_modules/verdaccio/bin/verdaccio -- -l host:port
```

**Comment by jachstet-sea** [comment]
I prefer "screen" for that purpose. It allows you to interactively start commands, then detach, logout and re-attach to that shell later if needed.

**Comment by 030** [comment]
What about a systemd script?

**Comment by viniciusbo** [comment]
@030 nice!

**Comment by 030** [comment]
What about deploying this file as ```/etc/systemd/system/verdaccio.service``` when npm install -g verdaccio is run so it is possible to start it as a service?

**Comment by jachstet-sea** [comment]
I oppose automatically creating a file below the /etc directory for one reason: Software installed using package managers (and I see npm as one) should be cleanly and completely removed when uninstalled using that package manager. If a file is copied to /etc using a hook (or did you think of another way?), you also need a hook that removes that file again when uninstalling.

**Comment by viniciusbo** [comment]
@jachstet-sea made a good point.

I think it should be optional also because people may prefer other daemon solutions. Altough most server's OSs use systemd by default, personally I prefer OpenRC (at my local machine).

**Comment by 030** [comment]
We could deploy it using [ansible](https://github.com/verdaccio/verdaccio/issues/84) for example

**Comment by juanpicado** [comment]
I think I can close this one, `systemd` patch has been already merged and `pm2` is also a good solution.

**Comment by juanpicado** [comment]
fixed on #89
